src/transaction.ts

```diff
@@ -216,6 +216,160 @@ export const sendTransaction = async function (
   return { transaction, confirmation }
 }
 
+/**
+ * Get all of the unamed resources used by the group in the given ATC
+ *
+ * @param algod The algod client to use for the simulation
+ * @param atc The ATC containing the txn group
+ * @returns The unnamed resources accessed by the group and by each transaction in the group
+ */
+export async function getUnnamedResourcesAccessed(algod: algosdk.Algodv2, atc: algosdk.AtomicTransactionComposer) {
+  const simReq = new algosdk.modelsv2.SimulateRequest({
+    txnGroups: [],
+    allowUnnamedResources: true,
+  })
+
+  const result = await atc.simulate(algod, simReq)
+
+  const groupResponse = result.simulateResponse.txnGroups[0]
+
+  if (groupResponse.failureMessage) {
+    throw Error(`Error during resource packing simulation in transaction ${groupResponse.failedAt}: ${groupResponse.failureMessage}`)
+  }
+
+  return {
+    group: groupResponse.unnamedResourcesAccessed,
+    txns: groupResponse.txnResults.map(
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      (t: any) => t.unnamedResourcesAccessed,
+    ) as algosdk.modelsv2.SimulateUnnamedResourcesAccessed[],
+  }
+}
+
+/**
+ * Take an existing ATC and return a new ATC with packed resources
+ *
+ * @param algod The algod client to use for the simulation
+ * @param atc The ATC containing the txn group
+ * @returns A new ATC with the resources packed into the transactions
+ */
+export async function packResources(algod: algosdk.Algodv2, atc: algosdk.AtomicTransactionComposer) {
```

Comment:
I assume implementation of `packResources` is gonna be replaced by call to an endpoint via algod? Any chance you can also add some todo notes on chunks that will only live temporarily until replaced with appropriate algod endpoint?

Reply:
Added a comment in [1d58463](https://github.com/algorandfoundation/algokit-utils-ts/pull/177/commits/1d5846351db3267f1574d6ff65e1f71fb7316776). Pretty much the entire function will be replaced once its in algod.